scripts/nats-bootstrap/functions.sh

```diff
@@ -4,6 +4,17 @@
 # - `Signing keys` are associated with accounts and can be 'templated' with pub/sub permissions and have a `role` name set.
 # - `Users` are applications connecting to NATS to pub/sub, users can be assigned a `role`.
 
+# While "base64" exists on MacOS, it seems to be an older verison without the '-w' option.
+# So we will use os_base64 to make it work cleanly on linux and macos
+if [ "$(uname)" == "Darwin" ]; then
+	os_base64=(gbase64)
```

Comment:
One thing I'd like to approach is detecting whether there's `gbase64` installed on a Mac. Otherwise we'll get new folks having weird errors because they are missing a utility. Something like testing the output of `which gbase64` against `""` should be sufficient. Then we can run that up front as part of a prerequisite check maybe?

Reply:
for sure.